commands/run.go

```diff
@@ -46,7 +46,7 @@ func Run(argv []string) {
 
 	// Parse args for command provider
 	subArguments, _ := docopt.Parse(
-		provider.Usage(), arguments["<args>"].([]string),
+		provider.Usage(), append([]string{cmd}, arguments["<args>"].([]string)...),
```

Comment:
Has taskcluster-worker been compiled and tested to work with this new sub command? I attempted to use it but it keeps printing out the help infromation and exiting with a status code of 1.  If I set the exit argument to false, then it seems to pass this line.